build/gen_stub.php

```diff
@@ -3894,25 +3900,23 @@ function generateFunctionEntries(?Name $className, array $funcInfos, ?string $co
 /**
  * @param iterable<FuncInfo> $funcInfos
  */
-function generateAttributeInitialization(iterable $funcInfos, ?string $parentCond = null): string {
+function generateAttributeInitialization(iterable $funcInfos, ?string $parentCond = null, iterable $allConstInfos): string {
     return generateCodeWithConditions(
         $funcInfos,
         "",
-        static function (FuncInfo $funcInfo) {
+        static function (FuncInfo $funcInfo) use ($allConstInfos) {
             $code = null;
 
             foreach ($funcInfo->args as $index => $arg) {
-                if (!$arg->isSensitive) {
-                    continue;
-                }
-
                 if ($funcInfo->name instanceof MethodName) {
                     $functionTable = "&class_entry->function_table";
                 } else {
                     $functionTable = "CG(function_table)";
                 }
 
-                $code .= "\tzend_mark_function_parameter_as_sensitive($functionTable, \"" . $funcInfo->name->getNameForAttributes() . "\", $index);\n";
+                foreach ($arg->attributes as $attribute) {
+                    $code .= $attribute->generateCode("zend_add_parameter_attribute(zend_hash_str_find_ptr($functionTable, \"" . $funcInfo->name->getNameForAttributes() . "\", strlen(\"" . $funcInfo->name->getNameForAttributes() . "\")), $index", "{$funcInfo->getArgInfoName()}_{$index}", $allConstInfos);
```

Comment:
```suggestion
                    $code .= $attribute->generateCode("zend_add_parameter_attribute(zend_hash_str_find_ptr($functionTable, \"" . $funcInfo->name->getNameForAttributes() . "\", strlen(\"" . $funcInfo->name->getNameForAttributes() . "\")), $index", "{$funcInfo->getArgInfoName()}_arg{$index}", $allConstInfos);
```
Clarify that it's arg X, a standalone number reads a bit weird in the arginfos